site/documentation/content/api/management/device-registry-v1.yaml

```diff
@@ -715,6 +717,25 @@ components:
             "period":
                $ref: '#/components/schemas/Period'
 
+      ConnectionDuration:
+         type: object
+         additionalProperties: false
+         required:
+            - effective-since
+         properties:
+            "effective-since":
+               type: string
+               format: date-time
+               description: The date-time on which the connection duration limit came into effect.
+            "max-seconds":
+               type: integer
+               default: -1
+               description: The maximum number of seconds to be allowed for a tenant for the
```

Comment:
I wonder if *seconds* is an appropriate unit here. What will be typical values?

Reply:
Instead of _seconds_, I think _minutes_ might be more appropriate unit here. For example AWS  describes connection time in terms of _minutes_ (2,250,000 minutes of connection for Free tier) & also in Hub service plans it is also defined in _minutes_.

Reply:
So, can we think of a reasonable use case where you would actually want to limit the overall connection time of all devices of a Tenant to something less than a minute? In the current Prometheus based implementation of the limits, we actually cache the data for more than a minute, don't we? So we usually wouldn't even be able to enforce such a limit properly. Personally, I therefore also believe that minutes should be more appropriate here ...